packages/trip-form/src/SubmodeSelector/index.tsx

```diff
@@ -53,12 +53,18 @@ export default function SubmodeSelector({
     : S.SubmodeSelector.Row;
 
   return (
-    <S.SubmodeSelector className={className} style={style}>
-      {label && <LabelType>{label}</LabelType>}
+    <S.SubmodeSelector
+      aria-label={label}
+      className={className}
+      role="group"
+      style={style}
+    >
+      {label && <LabelType aria-hidden="true">{label}</LabelType>}
```

Comment:
This label is unused by any input element, so I would convert to a span:
```suggestion
      {label && <LabelType aria-hidden="true" as="span">{label}</LabelType>}
```